docs/src/docs/welcome/integrations.mdx

```diff
@@ -37,10 +35,14 @@ For those who installed golangci-lint manually:
 Using it in an editor without `--fast-only` can freeze your editor.
 Golangci-lint automatically discovers `.golangci.yml` config for edited file: you don't need to configure it in VS Code settings.
 
-For those who want to install golangci-lint via extension:
-- install `golangci-lint-v2` via the `Go: Install/Update Tools` command after setting these configs.
+</details>
+
+<details>
+<summary style={{color: '#737380'}}>Recommended settings for installed golangci-lint via extension</summary>
+
+1. Install `golangci-lint-v2` via the `Go: Install/Update Tools` command after setting these configs.
   This will enable golangci-lint v1 to co-exist with v2.
-- and use the following settings:
+2. use the following settings:
```

Comment:
I assume the order is inverse: the configuration is the first, and the installation is the second.
`golangci-lint-v2` option doesn't appear in the tool list until it's configured as the lint-tool.